extensions/wikia/ExactTargetUpdates/client/ExactTargetResourceEnum.php

```diff
@@ -7,6 +7,8 @@ class ResourceEnum {
 	const CUSTOMER_KEY_USER_GROUPS = 'user_groups';
 	const CUSTOMER_KEY_USER_ID_WIKI_ID = 'UserID_WikiID';
 	const CUSTOMER_KEY_USER_PROPERTIES = 'user_properties';
+	const CUSTOMER_KEY_WIKI_CITY_LIST = 'city-list'; // TODO: Resolve prod key 'city_list'
+	const CUSTOMER_KEY_WIKI_CITY_CAT_MAPPING = 'city-cat-mapping'; // TODO: Resolve prod key 'city_cat_mapping'
```

Comment:
There are different keys for dev and prod env:
https://github.com/Wikia/app/blob/SUS-123-wiki-hooks/extensions/wikia/ExactTargetUpdates/tasks/ExactTargetWikiTaskHelper.php#L191
It's needed to be fixed before finishing this PR.